src/Math/babylon.math.ts

```diff
@@ -4244,15 +4215,18 @@ module BABYLON {
          */
         public updateFlag: number;
 
+        private readonly _m: Float32Array = new Float32Array(16);
+
         /**
-         * Gets or sets the internal data of the matrix
+         * Gets the internal data of the matrix
          */
-        public m: Float32Array = new Float32Array(16);
+        public get m(): Readonly<Float32Array> { return this._m; }

```

Comment:
To not break the API,  I made an accessor for that. Still, if its then modified, there while be a typescript error.